src/text/raw_text_writer.rs

```diff
@@ -148,7 +148,7 @@ pub struct RawTextWriter<W: Write> {
 /**
  * String escape codes, for Ion Clob.
  */
-fn string_escape_code_init() -> Vec<String> {
+pub(crate) fn string_escape_code_init() -> Vec<String> {
```

Comment:
Couple of thoughts here:
* This is now only used in the `IonValueFormatter`, so I think it makes sense for it to live in that module instead.
* I implemented this as a `Vec<String>` because it would only be constructed once per text writer. Now that it needs to exist for every `IonValueFormatter` (which we want to be very cheap to create), I think we should take the time to convert it to a `const STRING_ESCAPE_CODES: &'static [&'static str]`. It'll be a bit ugly as a literal, but then it'll be free to use/reference everywhere.